iroh-gateway/src/config.rs

```diff
@@ -65,16 +65,18 @@ impl Config {
 
     /// Derive server addr for non memory addrs.
     pub fn server_rpc_addr(&self) -> Result<Option<GatewayServerAddr>> {
+        #![allow(clippy::unreachable)]
```

Comment:
this doesnt help :/

Reply:
`#!` means apply to whole module, `#` means only next expression

Reply:
the warning was for `unreachable_patterns` which is a compiler lint not a clippy lint ( so no `clippy:` prefix)

Reply:
🙇